docs/src/user-guide/quick-start/overview.md

```diff
@@ -0,0 +1,151 @@
+# Overview
+
+This page will guide you through CLP's system requirements and help you choose which flavour of CLP will be best for you.
+
+If you already have CLP set up, click your flavour of CLP below to learn how to use it.
```

Comment:
```suggestion
This guide describes:

- [CLP's system requirements](#system-requirements);
- [how to choose a CLP flavor](#choosing-a-flavour); and
- [how to use CLP](#using-clp).
```

* I think this is faster to read and navigate.
* Let's add a new section at the bottom of the page called "Using CLP", and then move the cards you have below into that section.